pyproject.toml

```diff
@@ -36,6 +36,7 @@ dependencies = [
     "reasoning-gym @ git+https://github.com/open-thought/reasoning-gym.git",
     "tomli>=2.2.1",
     "pydantic[email]>=2.11.5",
+    "verifiers[all]",
```

Comment:
![screenshot-20250610-142716](https://github.com/user-attachments/assets/300a8cfe-7019-4a49-80d5-907e02f39215)
do we really need all ? seems a bit many depth